docs/Running-behind-nginx.md

```diff
@@ -1,8 +1,42 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx #
```

Comment:
Why did you add these hash tags after the heading? It's not done anywhere else and doesn't cause issues if we don't have them. I DO see the link in Netlify (left navbar) having a # now, which is bad. Not sure what else may be affected...

Reply:
I added them for consistency. 
I am used to adding the hash tags after headings from other guides I've written with markdown.

Removing them now so as to not cause issues.